cmd/podman/common/volumes.go

```diff
@@ -209,9 +209,29 @@ func getBindMount(args []string) (spec.Mount, error) {
 		switch kv[0] {
 		case "bind-nonrecursive":
 			newMount.Options = append(newMount.Options, "bind")
+		case "readonly", "read-only":
+			if setRORW {
+				return newMount, errors.Wrapf(optionArgError, "cannot pass 'readonly', 'ro', or 'rw' options more than once")
+			}
+			setRORW = true
+			switch len(kv) {
+			case 1:
+				newMount.Options = append(newMount.Options, "ro")
+			case 2:
+				switch strings.ToLower(kv[1]) {
+				case "true":
+					newMount.Options = append(newMount.Options, "ro")
+				case "false":
+					// RW is default, so do nothing
+				default:
+					return newMount, errors.Wrapf(optionArgError, "readonly must be set to true or false, instead received %q", kv[1])
+				}
+			default:
+				return newMount, errors.Wrapf(optionArgError, "badly formatted option %q", val)
+			}
 		case "ro", "rw":
 			if setRORW {
-				return newMount, errors.Wrapf(optionArgError, "cannot pass 'ro' or 'rw' options more than once")
+				return newMount, errors.Wrapf(optionArgError, "cannot pass 'readonly', 'ro', or 'rw' options more than once")
 			}
 			setRORW = true
 			// Can be formatted as one of:
```

Comment:
How about adding the following code in this block? So that we can reuse the code for handling `ro`. They are totally same.
```
if kv[0] == "readonly" {
	kv[0] = "ro"
}
```
Don't know if it is feasible.